### PR TITLE
x902 - Home page UX

### LINKED ISSUE #195: x902 - UX work on dashboard page
**User story**
As a PO (Minal) I would like to change the dashboard/home page to make it easier to use and understand for end users

**Who are the primary contacts for this story**
Minal

**Acceptance criteria**
- [] Horizontal scroll bar for the result table to be at the top of the table
- [] Add "Most Recent Operation" column after "Project", to contain the most recent operation column title.
- [] Change sorting order for status: active>paused>unstarted>completed>failed->withdrawn

### PR DESCRIPTION
**Pull request by BenTopping**
Closes #195 

- Added new column 'Most recent operation' to work progress
- Changed status sort order
- Made DataTable easier to use for large data sets

### CODE REVIEW

File: src/pages/WorkProgress.tsx [hunk 1]
```diff
@@ -200,23 +200,25 @@ const WorkProgress = ({ workTypes }: { workTypes: string[] }) => {
...
-                      {
-                        id: 'workNumber',
-                        desc: true
-                      }
-                    ]}
-                    columns={columns}
-                    data={searchResult.entries}
-                    ref={sortedTableDataRef}
-                  />
+                  <div className="max-h-[750px] overflow-y-auto overflow-x-auto">
```
Comment (seenanair) [inline src/pages/WorkProgress.tsx:203]: Is this 750 px  work for all screen sizes ? or do we need something like max-h-screen ?

File: src/types/stan.ts [hunk 2]
```diff
@@ -298,11 +298,11 @@ export type HistoryProps = HistoryStrProps | HistoryNumProps;
...
     WorkStatus.Active,
-    WorkStatus.Completed,
     WorkStatus.Paused,
-    WorkStatus.Failed,
-    WorkStatus.Withdrawn,
     WorkStatus.Unstarted,
+    WorkStatus.Completed,
+    WorkStatus.Failed,
+    WorkStatus.Withdrawn
   ];
```
Comment (seenanair) [inline src/types/stan.ts:306]: Please confirm the status state sort order

File: src/components/analysisLabware/analysisLabware.tsx [hunk 3]
```diff
@@ -185,7 +185,9 @@ export default function AnalysisLabware({ barcodes, comments, onChangeLabwareDat
       </Formik>
       {barcodes.length > 0 && (
         <motion.div initial={{ opacity: 0, y: -50 }} animate={{ opacity: 1, y: 0 }} className="mt-4">
-          <DataTable columns={columns} data={analysisLabwares} />
+          <div className="overflow-x-auto overflow-y-auto">
```
[the remark this answers is not in the document]
Comment (BenTopping) [reply src/components/analysisLabware/analysisLabware.tsx:188]: Yeah will have a look at doing that

File: src/pages/WorkProgress.tsx (hunk 1 above)
Comment (BenTopping) [reply src/pages/WorkProgress.tsx:203]: yeah max-h-screen is probably better than an arbitrary number

**Review by seenanair — APPROVED** [review]

**Comment by seenanair** [comment]
Looking great  👍

**Review by khelwood — APPROVED** [review]

**Pull request merged**